packages/mobile/src/screens/chat-screen/ChatScreen.tsx

```diff
@@ -168,7 +172,9 @@ export const ChatScreen = () => {
   const messageTop = useRef(0)
   const chatContainerTop = useRef(0)
   const chatContainerBottom = useRef(0)
+  const isKeyboardOpen = useSelector(getIsKeyboardOpen)
 
+  const hasTrack = useSelector(getHasTrack)
```

Comment:
Minor nit: At first read, this isn't clear to me that `hasTrack` means "there's a player control showing that we need to account for". Maybe either add a comment or rename the variable to make it more obvious?